api/v1/installation_types.go

```diff
@@ -93,6 +93,12 @@ type InstallationSpec struct {
 	// +optional
 	CalicoNetwork *CalicoNetworkSpec `json:"calicoNetwork,omitempty"`
 
+	// BPFBootstrapMode defines how the initial BPF networking configuration is set up.
+	// Default: Manual
+	// +optional
+	// +kubebuilder:validation:Enum=Auto;Manual
+	BPFBootstrapMode *BootstrapMode `json:"bpfBootstrapMode,omitempty"`
```

Comment:
I think this is too specific of configuration at too high level. Lets chat about this, I think at a minimum it should be nested in the CalicoNetworkSpec.